Backend/AMQPBackendDispatcher.php

```diff
@@ -79,6 +81,13 @@ public function getChannel()
      */
     public function getBackend($type)
     {
+        if (false === $this->backendsInitialized) {
```

Comment:
Don't compare two booleans to get a third, use directly `if (!$this->backendsInitialized)`